### PR TITLE
support swift3.0

### PR DESCRIPTION
**Pull request by huangboju**

### CODE REVIEW

File: Pod/Classes/JSSAlertView.swift [hunk 1]
```diff
@@ -181,19 +181,19 @@ public class JSSAlertView: UIViewController {
 		let titleString = titleLabel.text! as NSString
 		let titleAttr = [NSFontAttributeName:titleLabel.font]
```
Comment (tiferrei) [inline Pod/Classes/JSSAlertView.swift]: Hello there @huangboju, shouldn't this be: `let titleAttr = [NSFontAttributeName:titleLabel.font!] as [String: Any]`?

File: Pod/Classes/JSSAlertView.swift [hunk 2]
```diff
@@ -181,19 +181,19 @@ public class JSSAlertView: UIViewController {
...
-		self.titleLabel.frame = CGRect(x: self.padding, y: yPos, width: self.alertWidth - (self.padding*2), height: ceil(titleRect.size.height))
-		yPos += ceil(titleRect.size.height)
+		self.titleLabel.frame = CGRect(x: self.padding, y: yPos, width: self.alertWidth - (self.padding*2), height: ceil(titleRect.height))
+		yPos += ceil(titleRect.height)
 		
 		
 		// position text
 		if self.textView != nil {
 			let textString = textView.text! as NSString
 			let textAttr = [NSFontAttributeName:textView.font as! AnyObject]
```
Comment (tiferrei) [inline Pod/Classes/JSSAlertView.swift]: And this: let textAttr = [NSFontAttributeName: textView.font!] as [String: Any]?

File: Pod/Classes/JSSAlertView.swift [hunk 3]
```diff
@@ -181,19 +181,19 @@ public class JSSAlertView: UIViewController {
 		let titleString = titleLabel.text! as NSString
 		let titleAttr = [NSFontAttributeName:titleLabel.font]
 		let titleSize = CGSize(width: contentWidth, height: 90)
-		let titleRect = titleString.boundingRectWithSize(titleSize, options: NSStringDrawingOptions.UsesLineFragmentOrigin, attributes: titleAttr, context: nil)
```
Comment (syky27) [inline Pod/Classes/JSSAlertView.swift]: I am getting runtime crash on this line.

**Comment by syky27** [comment]
It is build passing but, I am still getting runtime exception on line 184

```
[_SwiftValue pointSize]: unrecognized selector sent to instance
```

File: Pod/Classes/JSSAlertView.swift (hunk 3 above)
Comment (huangboju) [inline Pod/Classes/JSSAlertView.swift]: I fixed it 
titleAttr it can't be [String: UIFont?], must is  [String: UIFont]

Comment (tiferrei) [inline Pod/Classes/JSSAlertView.swift]: Precisely, I also had the same runtime error, but it seems to work perfectly with `let titleAttr = [NSFontAttributeName:titleLabel.font!] as [String: Any]`

**Comment by syky27** [comment]
I've tested this and it seems to work. Thank you for contribution, I could not find a time for this too much on plate at work.

Thank you

**Pull request merged**